h2-engine/2.1.210.wso2v1/src/main/java/org/wso2/carbon/h2/osgi/console/H2DatabaseManager.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.h2.osgi.console;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.h2.Driver;
+import org.h2.engine.Database;
+import org.h2.engine.Engine;
+import org.wso2.carbon.h2.osgi.utils.H2Constants;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * H2 Database Manager.
+ */
+public class H2DatabaseManager {
+
+    private static H2DatabaseManager instance;
+    private static final Log log = LogFactory.getLog(H2DatabaseManager.class);
+
+    private H2DatabaseManager() {
+    }
+
+    public synchronized static H2DatabaseManager getInstance() {
+
+        if (instance == null)
```

Comment:
let's use Eager Instantiation for this

refer https://www.journaldev.com/1377/java-singleton-design-pattern-best-practices-examples#eager-initialization